app/services/renderer.py

```
@@ -1,680 +1,81 @@
-        else:
-            team_font = self.fonts["circuit_name"]
-            small_font = self.fonts["circuit_stats_value"]
-            driver_font = self.fonts["circuit_name"]
-            tech_font = self.fonts["circuit_location"]
-            stats_font = self.fonts["circuit_stats_value"]
-            points_font = self.fonts["circuit_stats_value"]
-
-        team_name, meta_text, team_pos, team_pts = build_team_header_values(team)
+    BLACK = 0
+    WHITE = 1
+    PALETTE = [BLACK, WHITE]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)